setup.py

```diff
@@ -104,14 +101,15 @@ def _set_manifest_path(manifest_dir: str, aggregate: bool = False, mapping: Mapp
                 continue  # avoid `lightning` -> `lightning/lightning`
             lines = [ln.replace(old, f"lightning/{new}") for ln in lines]
         lines = sorted(set(filter(lambda ln: not ln.strip().startswith("#"), lines)))
+        logging.debug(f"aggregated manifest consists of: {lines}")
         with open(manifest_path, mode="w") as fp:
             fp.writelines(lines)
     else:
         manifest_path = os.path.join(manifest_dir, "MANIFEST.in")
         assert os.path.exists(manifest_path)
     # avoid error: setup script specifies an absolute path
     manifest_path = os.path.relpath(manifest_path, _PATH_ROOT)
-    print("Set manifest path to", manifest_path)
```

Comment:
prints will appear if you `pip install -v`. This internal info shouldn't appear for users that are installing the package. With this I mean that it shouldn't use logging.info